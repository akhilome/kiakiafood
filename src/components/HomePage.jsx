import React from 'react';
import { Link } from 'react-router-dom';
import Nav from './Nav';

const HomePage = () => (
  <div>
    <Nav />
    <section className="hero">
      <h2>Quell Your Hunger in Minutes</h2>
      <Link to="/menu">
        <button type="button" className="btn-primary">
          Get Food ‚ü∂
        </button>
      </Link>
    </section>
  </div>
);

export default HomePage;
