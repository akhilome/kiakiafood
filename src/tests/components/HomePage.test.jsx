import React from 'react';
import HomePage from '../../components/HomePage';
import { shallowWrap } from '../helpers/contextWrapper';

describe('HomePage component', () => {
  it('should render homepage correctly', () => {
    const wrapper = shallowWrap(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should should button to get food', () => {
    const wrapper = shallowWrap(<HomePage />);
    const getFoodButton = wrapper.find('button');

    expect(getFoodButton.text()).toEqual('Get Food ‚ü∂');
  });
});
