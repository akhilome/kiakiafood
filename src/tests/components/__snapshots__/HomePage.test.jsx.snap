// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render homepage correctly 1`] = `
<section
  className="hero"
>
  <h2>
    Quell Your Hunger in Minutes
  </h2>
  <Link
    replace={false}
    to="/menu"
  >
    <button
      className="btn-primary"
      type="button"
    >
      Get Food ‚ü∂
    </button>
  </Link>
</section>
`;
